File: src/main/java/com/senderman/lastkatkabot/command/user/BncHelp.java
package com.senderman.lastkatkabot.command.user;

import com.annimon.tgbotsmodule.commands.context.MessageContext;
import com.senderman.lastkatkabot.command.CommandExecutor;
import com.senderman.lastkatkabot.config.BotConfig;
import org.springframework.stereotype.Component;

@Component
public class BncHelp implements CommandExecutor {

    private final String bncPhotoId;

    public BncHelp(BotConfig config) {
        this.bncPhotoId = config.bncHelpPictureId();
    }

    @Override
    public String command() {
        return "/bnchelp";
    }

    @Override
    public String getDescription() {
        return "помощь по игре Быки и Коровы";
    }

    @Override
    public void accept(MessageContext ctx) {
        ctx.replyWithPhoto()
                .setFile(bncPhotoId)
                .callAsync(ctx.sender);
    }
}

File: src/main/kotlin/com/senderman/lastkatkabot/bnc/commands/BNCHelp.kt
package com.senderman.lastkatkabot.bnc.commands

import com.annimon.tgbotsmodule.api.methods.Methods
import com.senderman.lastkatkabot.LastkatkaBotHandler
import com.senderman.lastkatkabot.Services
import com.senderman.neblib.CommandExecutor
import org.telegram.telegrambots.meta.api.objects.Message

class BNCHelp(private val handler: LastkatkaBotHandler) : CommandExecutor {
    override val command: String
        get() = "/bnchelp"
    override val desc: String
        get() = "помощь по игре Быки и Коровы"

    override fun execute(message: Message) {
        val sendPhoto = Methods.sendPhoto()
            .setChatId(message.chatId)
            .setFile(Services.botConfig.bncphoto)
        if (message.isReply) sendPhoto.replyToMessageId =
            message.replyToMessage.messageId else sendPhoto.replyToMessageId = message.messageId
        sendPhoto.call(handler)
    }
}